### PR TITLE
✨ GATK HC CRAM to gVCF Standalone

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

This PR creates a modern GATK HaplotypeCaller CRAM to gVCF workflow. Some features of the workflow:
- Takes the same reference tar as the alignment workflow
- Indexes the dbsnp file if the user does not provide an index file
- Converts the input CRAM into a BAM
- Conditionally checks the BAM contamination if the user does not provide a value in the inputs
- Renames the same and collects metrics

Breakdown of other changes:
-  picard_mergevcfs_python_renamesample.cwl: Inputbinding fixed so that INPUTs would now end up in the proper place of the MergeVCFs command
- created a new verifybamid_contamination_conditional.cwl. This tool works the same as the previous verifybamid_contamination.cwl except it has a new optional input for a contamination value and the contamination file inputs are optional. Functionally, if the user provides the contamination value it will not run the script, even if the contamination file inputs are provided
- Readme updated

Closes https://github.com/d3b-center/bixu-tracker/issues/721

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Cavatica contamination value + contamination files: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/70210306-a124-4fa9-ab58-c4f61e9bb015/
- [x] Cavatica contamination value only: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/6a85490f-8a59-4ab1-bb1a-547a4d7e4b96/#
- [x] Cavatica contamination value of 0: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/3b08d3c5-0dd4-44f5-bdc1-c83afee6bfd6/#
- [x] Cavatica contamination files only: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/de4fb8bd-20e4-41d2-8fbb-13eb08e1d6cf/#
- [x] Workflow passes CWLtool validation
- [x] The new workflow outputs a contamination value of `2.4251866666666667E-5`. This value matches exactly what was generated in the original harmonization process used to generate the CRAM (and ultimately its gVCF) here: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/e57126b9-d033-4642-8a7f-a0fd606406f3/stats/

**Test Configuration**:
* Environment: Cavatica, cwltool 3.0.20200324120055
* Test files: See Cavatica test

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by migbro** [comment]
Hmmm, something seems. bit off. In giving the precalc value, you got `-contamination 0.009684066666666666 ` as what haplotype caller ends up running. When you run verifyBamID, you get `-contamination 2.4251866666666667E-5 `.

**Comment by dmiller15** [comment]
That precalculated value I handed in was a value from a different input here: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/6b6d3fed-f29f-4b26-8299-86549409336e/. I ran them concurrently. Wasn't too concerned about making nice data, just proving it worked.

File: workflows/kfdrc_gatk_haplotypecaller_wf.cwl [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      input_vcf: picard_mergevcfs_python_renamesample/output
+      reference_dict: untar_reference/dict
+      wgs_evaluation_interval_list: wgs_evaluation_interval_list
+    out: [output]
+
+$namespaces:
+  sbg: https://sevenbridges.com
+hints:
+  - class: 'sbg:AWSInstanceType'
+    value: c4.8xlarge;ebs-gp2;400
```
Comment (migbro) [inline workflows/kfdrc_gatk_haplotypecaller_wf.cwl:140]: One small request, make the value `c5.9xlarge;ebs-gp2;850`

Comment (dmiller15) [reply workflows/kfdrc_gatk_haplotypecaller_wf.cwl]: Done

**Pull request merged**